src/nomad_simulations/schema_packages/numerical_settings.py

```diff
@@ -309,14 +310,27 @@ class KMesh(Mesh):
     """
 
     label = Quantity(
-        type=MEnum('k-mesh', 'q-mesh'),
+        type=MEnum('k-mesh', 'q-mesh'),  # ? add g-mesh for LAPW
```

Comment:
go ahead, this is a simple identifier and g-mesh is a good idea

Reply:
The description seems a bit too specific for linear response, I'd see to extend that.